basic/src/test/java/org/jboss/hal/testsuite/test/configuration/undertow/UndertowTestCaseAbstract.java

```diff
@@ -62,6 +64,12 @@ public static void mainTearDown() throws InterruptedException, IOException, Time
         }
     }
 
+    @After
+    public void disposeOfReloadRequiredWindow() throws InterruptedException, TimeoutException, IOException {
+        Console.withBrowser(browser).dismissReloadRequiredWindowIfPresent();
+        administration.reloadIfRequired();
```

Comment:
IMO `administration.reloadIfRequired();` should be sufficient and `Console.withBrowser(browser).dismissReloadRequiredWindowIfPresent();` is not necessary since console is refreshed before each Undertow config test.